### ISSUE TITLE
Blue bar present when running in Android 2.2 but not when in Android 4.0

### ISSUE THREAD

**Issue by ruiaraujo** [JakeWharton/ActionBarSherlock#276]
Using the same program with the same resources ( this means styles and drawables ) I get differente results when running in Android 2.2 and Android 4.0.

The most annoying of them all is definitely a blue bar under the ActionBar.

You can see the screenshots here:

![](http://i.imgur.com/uSSyI.png)
![](http://i.imgur.com/GJsID.png)

Edit: I forgot to add that I was using the 4.0 RC version.

**Comment by JakeWharton** [comment]
Are you specifying styles for both the native and compatibility action bar?

Something like this:

``` xml
<style name="Theme.Styled" parent="Theme.Sherlock.Light.DarkActionBar">
    <item name="actionBarStyle">@style/Widget.Styled.ActionBar</item>
    <item name="android:actionBarStyle">@style/Widget.Styled.ActionBar</item>
</style>
<style name="Widget.Styled.ActionBar" parent="Widget.Sherlock.Light.ActionBar.Solid.Inverse">
    <item name="background">@drawable/bg_striped</item>
    <item name="android:background">@drawable/bg_striped</item>
</style>
```

**Comment by jgilfelt** [comment]
This was my experience as well. Native `Theme.Holo.Light` uses the solid style ActionBar by default, whereas `Theme.Shelock.Light` uses the transparent one with the blue keyline.

Easy enough to workaround by specifying an `actionBarStyle`, but I think it's a bug none the less.

**Comment by JakeWharton** [comment]
Fixed in `4.0-wip`. Thanks.
